library/md.c

```diff
@@ -449,7 +449,8 @@ unsigned char mbedtls_md_get_size( const mbedtls_md_info_t *md_info )
     if( md_info == NULL )
         return( 0 );
 
-    return md_info->size;
+    /* NOTE: function should be updated if the value of "size" may exceed 255. */
```

Comment:
Please suggest a better solution, if possible.

This function doesn't have a way to return indicate an error, I couldn't find any way of at logging an error in case *md_info*->*size* was too large and I haven't seen any assertions in the code as well, so I'm guessing adding one here would be against mbedTLS coding standards.